doc/user/content/ingest-data/mysql/amazon-rds.md

```diff
@@ -150,6 +150,102 @@ to connect:
 
 {{< /tab >}}
 
+{{< tab "Use AWS PrivateLink">}}
+
+[AWS PrivateLink](https://aws.amazon.com/privatelink/) lets you connect
+Materialize to your RDS instance without exposing traffic to the public
+internet. To use AWS PrivateLink, you create a network load balancer in the
+same VPC as your RDS instance and a VPC endpoint service that Materialize
+connects to. The VPC endpoint service then routes requests from Materialize to
+RDS via the network load balancer.
+
+{{< note >}}
+Materialize provides a Terraform module that automates the creation and
+configuration of AWS resources for a PrivateLink connection. For more details,
+see the [Terraform module repository](https://github.com/MaterializeInc/terraform-aws-rds-privatelink).
```

Comment:
QQ: Do we need to mention that since the repo uses PostgreSQL as an example, they should update for MySQL?

Reply:
And maybe blurb the same in the privatelink repo as well?

Reply:
Ah ... never mind.  I just saw your helpful link to the issue w.r.t. this.